File: problemas/iniciante/1759/1759.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let n: i32 = input.trim().parse().unwrap();

    let mut resposta: String = "Ho".to_owned();
    for _ in 1..n {
        resposta.push_str(" Ho");
    }
    resposta.push_str("!");

    println!("{}", resposta);
}

File: problemas/iniciante/1759/1759.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let N = parseInt(lines.shift());

--N;
let resposta = [];
for (let i = 0; i < N; ++i) {
    resposta.push("Ho");
}
resposta.push("Ho!");

console.log(`${resposta.join(' ')}`);